deltachat-jsonrpc/src/api.rs

```diff
@@ -2157,12 +2157,14 @@ impl CommandApi {
 
     // mimics the old desktop call, will get replaced with something better in the composer rewrite,
     // the better version will just be sending the current draft, though there will be probably something similar with more options to this for the corner cases like setting a marker on the map
+    #[allow(clippy::too_many_arguments)]
     async fn misc_send_msg(
```

Comment:
Not sure whether it's necessary to change this function and `misc_set_draft()` since it's going to be replaced anyway?

Is this function and `misc_set_draft()` still in use in the Desktop UI?